db/migrate/20240205173114_create_greetings.rb

```diff
@@ -0,0 +1,9 @@
+class CreateGreetings < ActiveRecord::Migration[7.1]
+  def change
+    create_table :greetings do |t|
+      t.string :content
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Well done creating the greeting table. Kindly create 5 greetings in the seed file. Otherwise, when users open the react app they should create greetings by themselves on the side of the rail [requirements.](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/hello_world_two_apps.md)